### DISCUSSION TITLE
Custom Radio

### DISCUSSION THREAD

**Discussion by richbachman** [ariakit/ariakit#2185] [Questions]
Hi Ariakit team.

I'm curious a custom Radio is possible, similar to the [custom Checkbox example](https://ariakit.org/examples/checkbox-custom)?

Thanks!

**Comment by diegohaz** [comment]
Yes, I guess it would be similar to the checkbox example. Would you like to work on an [example](https://github.com/ariakit/ariakit/issues/939)?

**Reply by richbachman** [reply]
@diegohaz yeah I'm happy to take that on. I'll let you know if I need any assistance.

**Reply by richbachman** [reply]
@diegohaz sorry for the delay. I finally had some time to work on a custom radio example. Here's the [PR](https://github.com/ariakit/ariakit/pull/2218).
